sdk/all/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdk.java

```diff
@@ -97,7 +126,7 @@ public Tracer get(String instrumentationName, String instrumentationVersion) {
       return delegate.get(instrumentationName, instrumentationVersion);
     }
 
-    public TracerProvider unobfuscate() {
+    public SdkTracerProvider unobfuscate() {
```

Comment:
@trask @anuraaga does this have any implications for the agent's usage of the obfuscated pattern?

Reply:
Agent doesn't use this anymore :)